CMakeLists.txt

```diff
@@ -155,6 +155,8 @@ set (SOURCE_FILES_PUT_GET
         cpp/StorageClientFactory.hpp
         cpp/StorageClientFactory.cpp
         cpp/RemoteStorageRequestOutcome.hpp
+        cpp/entities.cpp
+        cpp/entities.hpp
```

Comment:
Since this is copied from ODBC I think better to make ODBC to reuse this later. Can we move entities.hpp to include/snowflake or move the function declaration into an existing header file there?